app/connectors/v3/getFinancialDetails/GetFinancialDetailsConnector.scala

```diff
@@ -23,9 +23,9 @@ import uk.gov.hmrc.http.{HeaderCarrier, HttpClient}
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class FinancialDetailsConnector @Inject()(httpClient: HttpClient,
-                                          appConfig: AppConfig)
-                                         (implicit ec: ExecutionContext) {
+class GetFinancialDetailsConnector @Inject()(httpClient: HttpClient,
+                                             appConfig: AppConfig)
+                                            (implicit ec: ExecutionContext) {
 
   def getFinancialDetails(vatcUrl: String)(implicit hc: HeaderCarrier): Future[GetFinancialDetailsResponse] = {
     httpClient.GET[GetFinancialDetailsResponse](url = appConfig.getFinancialDetailsUrl + vatcUrl)
```

Comment:
Probably worth renaming the parameter here to `vrn`. There is some missing items from the url, see API 1811 spec

Reply:
Probably worth adding it in AppConfig. I believe the params should be:
includeLocks = false
includeStatistical = true
onlyOpenItems = false
calculateAccruedInterest = true
removePOA = false
customerPaymentInformation = false
dateFrom = LocalDate.now().minusYears(2)
dateTo = LocalDate.now()

For includeLocks, includeStatistical, onlyOpenItems, calculateAccruedInterest, removePOA and customerPaymentInformation it may be worth adding them in application.conf just in case our assumption is wrong or more data is needed later.

For dateFrom and dateTo, you may need to update DateHelper to have a method for LocalDate.now

Also worth adding headers as well see: https://github.com/hmrc/penalties/blob/855be3e9dccb6de482d6a22ad4523bba7635dead/app/connectors/v3/getPenaltyDetails/GetPenaltyDetailsConnector.scala#L33